_posts/2017-11-16-totw-65.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Tip of the Week #65: Putting Things in their Place"
+layout: tips
+sidenav: side-nav-tips.html
+published: false
+permalink: tips/65
+type: markdown
+order: "065"
+---
+
+Originally posted as totw/65 on 2013-12-12
+
+By Hyrum Wright (hyrum@hyrumwright.org)
```

Comment:
Enclose email in mailto link:

[(hyrum@hyrumwright.org)](mailto:hyrum@hyrumwright.org)

(I know the mailto: is not as often used anymore, but the styling is also nice to have.)